Jenkinsfile-e2e

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env groovy
+
+def nodeImage = "329802642264.dkr.ecr.eu-west-1.amazonaws.com/bbc-news/node-10-lts:10.16.0-1"
+def slackChannel = '#si_repo-simorgh'
+
+// Run latest every 4 hours
+def cron_string = env.BRANCH_NAME == "latest" ? "H 0 */4 * * *" : ""
+
+def notifySlack(messageParameters) {
+  def message = "${messageParameters.emoji} Simorgh E2Es ${messageParameters.buildStatus} on \"${messageParameters.branchName}\" [build #${env.BUILD_NUMBER}] ${env.BUILD_URL}"
+
+  slackSend(
+    channel: messageParameters.slackChannel,
+    color: messageParameters.colour,
+    message: message
+  )
+}
+
+def stageHasFailed = false;
+
+node {
+  properties([pipelineTriggers([cron(cron_string)])])
+  timeout(time: 90, unit: 'MINUTES') {
+    withEnv([
+      'CI=true'
+    ]) {
+      cleanWs() // Clean the workspace
+      checkout scm // Checkout from git
+      docker.image("${nodeImage}").inside {
+        try {
+          stage ('Install & Setup') {
+            error('simulated error')
+            sh 'make install'
+          }
+          stage ('Running E2Es across environments') {
+            parallel (
+              'Local Prod Tests': {
+                try {
+                  sh 'CYPRESS_SMOKE=false npm run build && xvfb-run -a npm run test:prod:ci;'
```

Comment:
Running this command from the makefile always failed to start the server `xvfb-run` would fail.